### ISSUE TITLE
Accessibility : Data tables are not speaking their row and column header text to screen reader users

### ISSUE THREAD

**Issue by HarshalCapgemini** [xamarin/Xamarin.Forms#14854]
Accessibility for Android

Data tables are not speaking their row and column header text to screen reader users when they move through the cells.
Code as an accessible data table with row and column headers and make sure the screen reader user hears the header spoken when they move between rows and columns in the cells. 
 
https://pauljadam.com/guides/tables.html

android:accessibilityHeading = "true" similar property is needed in Xamarin Forms.

**Comment by jfversluis** [comment]
@rachelkang ? Don't want to be the person that just passed everything accessibility to you, but... 😬

**Comment by rachelkang** [comment]
Hi, @HarshalCapgemini - thanks for the feedback!

We've actually introduced a new API for headings in .NET MAUI and in Xamarin Community Toolkit. So as long as you add the XCT nuget, you should be able to apply this new property `SemanticEffect.HeadingLevel` to controls in your Xamarin.Forms project. 

You can learn more about this property and the other SemanticEffect APIs in XCT [here](https://github.com/xamarin/XamarinCommunityToolkit/blob/2c267757be8aa4e6b4ce8371592628950af0572f/src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Semantic/SemanticEffect.shared.cs#L12). Let me know if you have any further questions! If the HeadingLevel property isn't working you, seeing a repro of the issue would be helpful for us to look into this further.

**Comment by jfversluis** [comment]
Didn't hear back so I'll assume our information helped you get past this, thanks!
